smithy-typescript-ssdk-libs/package.json

```diff
@@ -5,14 +5,11 @@
   "description": "Smithy TypeScript SSDK packages",
   "main": "index.js",
   "scripts": {
-    "clean": "yarn clear-build-cache && yarn clear-build-info && lerna clean -y",
-    "clear-build-cache": "rimraf ./*/dist ./*/types, ./*/*.tgz",
-    "clear-build-info": "rimraf **/*.tsbuildinfo",
-    "build": "lerna run build",
-    "pretest": "yarn build",
-    "test": "jest",
-    "lint": "yarn run eslint **/src/**/*.ts",
-    "format": "prettier --write **/*.{ts,js,md,json}"
+    "clean": "cd server-apigateway && yarn clean && cd ../server-common && yarn clean",
```

Comment:
is `turbo run clean --force` an option?

Reply:
it does seem to work, though I swear I tried it last week and it did not. Changed it to use this